### DISCUSSION TITLE
Example 'streams-generator-spdif' not working on ESP8266

### DISCUSSION THREAD

**Discussion by joergber** [pschatzmann/arduino-audio-tools#567] [Q&A] [answered]
Hi,

I am trying to use SPDIF on an ESP8266 by using the 'streams-generator-spdif' example.
But I am not able to get it  to work neither on pin RX nor D4.
It's working fine on a ESP32 (RX-pin).

Can somebody pls. tell me what I am missing?

THX!
Jörg

**Comment by pschatzmann** [comment, marked as answer]
SPDIF requires very high sample rates. I guess the ESP8266 just can't keep up...

**Reply by joergber** [reply]
But it seems to work with ESP8266Audio.
I have not tested it, yet ...
